### PR TITLE
Couple of formatting/grammar fixes

### PR DESCRIPTION
**Pull request by diegopetrucci**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,12 +27,12 @@ When building a `Box`, all you need to care about are `Sections`s and `Node`s.
...
-                |---+ RowId.user <> IconTitleDetailsComponent(icon: image, title: patient.name)
-                |-+ Section(id: SectionId.consultantDate,
-                            header: EmptySpaceComponent(height: 24, color: .clear))
-                |---+ RowId.loading <> LoadingIndicatorComponent(isLoading: true)
+|-+ Section(id: SectionId.user,
+header: EmptySpaceComponent(height: 24, color: .clear))
+|---+ RowId.user <> IconTitleDetailsComponent(icon: image, title: patient.name)
+|-+ Section(id: SectionId.consultantDate,
+header: EmptySpaceComponent(height: 24, color: .clear))
+|---+ RowId.loading <> LoadingIndicatorComponent(isLoading: true)
```
Comment (sergdort) [inline README.md]: It does not look nice 😜
screen shot 2018-05-08 at 11 43 36

**Comment by diegopetrucci** [comment]
@sergdort @mluisbrown Xcode just dropped all formatting ¯\_(ツ)_/¯
It is fixed now.

**Comment by mluisbrown** [comment]
@diegopetrucci should have used VS Code for editing Markdown 😛

File: README.md [hunk 2]
```diff
@@ -131,7 +131,13 @@ let bento = Box.empty // 3
 	|---+ RowId.id <> Component() // 1
 ```
 
-As you can see, `<>` has a `ComposingPrecedence`, `|---+` has a `NodeConcatenationPrecedence `, which is higher then `|-+`, `SectionConcatenationPrecedence`, which means that Nodes will be computed first. The order of the expression above is:
+As you might have noticed:
+* <> has ComposingPrecedence;
```
Comment (mluisbrown) [inline README.md]: `<>` and `ComposingPrecedence` should be code formatted (with backticks)

File: README.md [hunk 3]
```diff
@@ -131,7 +131,13 @@ let bento = Box.empty // 3
...
+* |---+ has NodeConcatenationPrecedence
```
Comment (mluisbrown) [inline README.md]: Ditto.

**Pull request merged**